crates/tokmd-content/CLAUDE.md

```
@@ -44,7 +44,9 @@ pub fn count_tags(text: &str, tags: &[&str]) -> Vec<(String, usize)>
 pub fn entropy_bits_per_byte(bytes: &[u8]) -> f32
 ```
 
-## Entropy Calculation
+## Implementation Details
+
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)